src/main/java/edu/cornell/kfs/concur/service/impl/ConcurAccessTokenServiceImpl.java

```diff
@@ -97,38 +90,47 @@ protected String buildCredentialsStringForRequestingNewAccessToken(String loginU
     @Transactional
     @Override
     public void refreshAccessToken() {
-        AccessTokenDTO refreshedToken = buildRefreshAccessTokenOutput();
+        AccessTokenDTO refreshedToken = callConcurEndpoint(
+                this::callRefreshAccessTokenEndpoint, AccessTokenDTO.class);
```

Comment:
Similar thought on adding clarity, move "this::CallRequestAccessTokenEndpoint" to a variable: Function<Client, Response> endpointCaller